src/reaper_osara.cpp

```diff
@@ -3756,6 +3780,58 @@ void cmdMoveToPrevItem(Command* command) {
 	}
 }
 
+// REAPER's stock actions adjust horizontal zoom in increments of 20%, returning pixels per second.
+// We provide stepped zoom settings and report expected behaviour in time per keypress instead. It's easier to understand non-visually.
+void reportZoomStepsAsTime(double time) {
+	TimeFormat tf = TF_MILSEC;
+	if (time >= 1.0) tf = TF_ROUNDSEC;
+	// TRANSLATORS: Reported when users change horizontal zoom in steps.
+	// {} will be replaced with a formatted time increment (e.g. "1 ms", "2 sec").
+	outputMessage(format(translate("{} per keypress"), formatTime(time, tf)));
+}
+
+void zoomSteps(bool zoomOut) {
```

Comment:
Rename zoomSteps to adjustZoomByStep.
```suggestion
void adjustZoomByStep(bool zoomOut) {
```